javascript/javascript2/week1/index2js2week1.html

```diff
@@ -0,0 +1,79 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="main.css" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+    rel="stylesheet"
+  />
+</head>
+<body>
+  <main>
+    <header>
+      <div class="wrapper">
+        <h1>HyfBay</h1>
+        <h2>- Buy the okay'est products</h2>
+      </div>
+    </header>
+    <section class="table-view-control">
+      <div class="search">
+        <label>What are you looking for?</label>
```

Comment:
You should specify the elements that this label belongs to, so `<label for="searchName">...</label>` in your case. 
You can alsowrap input in the label element like this `<label><input/></label>` but the first way is probably better. 
Doing that makes you page more accessible, for example for people who use screen readers.  Label element is not only visual - browser should know wich element is labeled speciffically.